spec/unit/matrixrtc/RTCEncrytionManager.spec.ts

```diff
@@ -428,6 +429,94 @@ describe("RTCEncryptionManager", () => {
                 "@carol:example.org:CAROLDEVICE",
             );
         });
+
+        it("Should store keys for later retrieval", async () => {
+            jest.useFakeTimers();
+
+            const members = [
+                aCallMembership("@bob:example.org", "BOBDEVICE"),
+                aCallMembership("@bob:example.org", "BOBDEVICE2"),
+                aCallMembership("@carl:example.org", "CARLDEVICE"),
+            ];
+            getMembershipMock.mockReturnValue(members);
+
+            // Let's join
+            encryptionManager.join(undefined);
+            encryptionManager.onMembershipsUpdate(members);
+
+            await jest.advanceTimersByTimeAsync(10);
+
+            // Simulate Carl leaving then joining back, and key received out of order
```

Comment:
Need some help understanding: If this test just emits a key at index 0 and then index 5 for Carl, what part of this represents the keys being received "out of order"?